### PR TITLE
adds doc for repo publishing

### PR DESCRIPTION
**Pull request by tannerjfco**
## The Problem:
drud/general#33 OP - Add guidelines and preparation steps for publishing repositories to the public org space.
## Related Issue Link(s):
drud/general#33 OP

### CODE REVIEW

**Review by cyberswat — COMMENTED** [review]
I think this is a great start.  Is it safe to assume that there will be follow-up documentation that helps us understand how to deal with repositories that met this criteria when opened, but no longer represent the points contained in this publishing guide?

File: development/publishing_repositories.md
Comment (rickmanelius) [inline development/publishing_repositories.md:17]: To address @cyberswat's question on what happens when a repo that was made public no longer meets the guidelines, I suggest adding the following.

"Occasionally, a repository opened to the public may no longer meet the aforementioned guidelines. In these situations, the preference will be to keep the repository open while providing additional information on the main project page indicating any change in support, maintenance, etc. If a repository needs to be pulled back private, the team will use its best judgment in how to do so without disrupting users that may be using and/or dependent upon it."

**Review by rickmanelius — CHANGES_REQUESTED** [review]
I added one suggestion to address @cyberswat's question. After that, this has my approval!

File: development/publishing_repositories.md
Comment (rfay) [reply development/publishing_repositories.md:17]: And of course, we now have a third option for public repos we don't maintain or don't meet our standards: Move them to drud-archive.

Comment (rickmanelius) [reply development/publishing_repositories.md:17]: I was going to reference that as well. I only left it out because, given that redirects, this change should be relatively transparent to the end-user. But happy to add that in as well!

**Review by cyberswat — APPROVED** [review]

**Comment by tannerjfco** [comment]
I've added the suggestion under a heading for retiring public repos, with the slight change to say "If a repository needs to be pulled back private or archived"

**Review by rickmanelius — APPROVED** [review]
Thanks!

**Comment by rickmanelius** [comment]
Two approves. Merged!

**Pull request merged**
